ibm/mas_devops/roles/gencfg_jdbc/README.md

```diff
@@ -0,0 +1,45 @@
+suite_config
+============
+
+This role is used to configure db in Maximo Application Suite.
+
+Role Variables
+--------------
+
+ole Variables
```

Comment:
double pasted "ole Variables"? Line needs to be removed

Reply:
removed , Thanks